data-prepper-plugins/rds-source/src/main/java/org/opensearch/dataprepper/plugins/source/rds/model/DbTableMetadata.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.source.rds.model;
+
+import java.util.Map;
+
+public class DbTableMetadata {
+
+    private static final String DB_METADATA_KEY = "dbMetadata";
+    private static final String TABLE_COLUMN_METADATA_KEY = "tableColumnDataTypeMap";
+    private final DbMetadata dbMetadata;
+    private final Map<String, Map<String, String>> tableColumnDataTypeMap;
```

Comment:
Can you clarify the key and value in this map? Maybe add a javadoc?

Reply:
It's Map of table name to table column data type map. e.g. `{ "table1" : { "column1" : "int", "column2" : "varchar" }, "table2" : { "column1" : "int" } }`

Will add java doc